src/utils.ts

```diff
@@ -1,5 +1,11 @@
 import React from 'react';
 
+function decodeHtmlEntities(text: string): string {
+  const textarea = document.createElement('textarea');
```

Comment:
this would break in SSR because of the use of `document` on the server

Reply:
Is there an SSR-friendly way to `decodeHTML` entities on React? Maybe the textarea hack is not the best option here

Reply:
I solved it with a map of entities

Reply:
while you could use the DOMParser API on client side like this

```ts
function decodeHtmlEntities(str) {
    const parser = new DOMParser();
    const doc = parser.parseFromString(str, 'text/html');
    return doc.documentElement.textContent;
}

const encodedStr = "&quot;Hello, World!&quot;";
const decodedStr = decodeHtmlEntities(encodedStr);
```

 you would still have to do this on server side right?
 in that case there are a few parsers that would do the trick
 
 would the decoding be necessary the pure text anyway? 😅
 just asking because the JSON repsonse for one of the textnodes lookes like this for me and once it enters the DOM it looks just fine
 
> `hello world's most \"awesome\" text`

Reply:
I think the current solution with a static map works perfectly fine, I honestly would not go for third party libraries for this task